Userland/Libraries/LibWeb/HTML/HTMLScriptElement.cpp

```diff
@@ -127,8 +128,8 @@ void HTMLScriptElement::execute_script()
         // 2. Run the module script given by el's result.
         (void)verify_cast<JavaScriptModuleScript>(*m_result.get<JS::NonnullGCPtr<Script>>()).run();
     } else if (m_script_type == ScriptType::ImportMap) {
-        // FIXME: 1. Register an import map given el's relevant global object and el's result.
-        dbgln("FIXME: HTMLScriptElement import map support");
+        // FIXME: Verify the current import map is empty per https://html.spec.whatwg.org/multipage/webappapis.html#import-map-parse-results "register an import map"
```

Comment:
The original comment was a direct quote from the spec. Please leave it in! (Without the `FIXME` part of course.)